WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -75,7 +75,7 @@ @implementation JetpackSettingsViewController
 
 - (instancetype)initWithBlog:(Blog *)blog
 {
-    self = [super init];
+    self = [super initWithNibName:nil bundle:nil];
```

Comment:
I just realized this is calling `super`, but if we treat it like a convenience initializer it should probably be calling `self`.